components/search/SearchInterface.vue

```diff
@@ -1,4 +1,4 @@
-<template>
+defaultFacetNames<template>
```

Comment:
what's 'defaultFacetNames' doing here?

Reply:
It's an unwanted guest and has been duly evicted.